sympy/polys/matrices/tests/test_lll.py

```diff
@@ -0,0 +1,32 @@
+from sympy.core.numbers import Rational
+from sympy.external.gmpy import HAS_GMPY
+from sympy.polys.domains import ZZ_gmpy
+from sympy.polys.domains.groundtypes import GMPYInteger
+from sympy.polys.matrices import lll, DomainMatrix
+
+int_class = GMPYInteger if HAS_GMPY else int
```

Comment:
It isn't necessary to do this. Just use ZZ: it will give the correct type whether gmpy is installed or not.

https://docs.sympy.org/latest/modules/polys/domainsintro.html

Reply:
Ok I will try that, thanks.